src/Tizen.Nlp/Tizen.Nlp/NaturalLanguageProcess.cs

```diff
@@ -0,0 +1,251 @@
+/*
+* Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Tizen.Applications;
+
+namespace Tizen.Nlp
+{
+
+    /// <summary>
+    /// This class contains the methods and inner class related to the NLP processing.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public class NaturalLanguageProcess
+    {
+        /// <summary>
+        /// An EventHandler to receive data after wordtokenize processing.
+        /// </summary>
+        /// <remarks>
+        /// if this succeeds, the object contain Key/Pari about Nlp will be received.
+        /// and you can use TryGetValue() with the Key "token" or "tag" to get a array of string for access,
+        /// </remarks>
+        public event EventHandler<MessageReceivedEventArgs> WordTokenizeResultReceived;
```

Comment:
Provide as async method.  it is not event. it is a result of request.

We prefer task based asynchronous method
https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap